File: 1967-number-of-strings-that-appear-as-substrings-in-word/1967-number-of-strings-that-appear-as-substrings-in-word.py
class Solution(object):
    def numOfStrings(self, patterns, word):
        """
        :type patterns: List[str]
        :type word: str
        :rtype: int
        """
        res = 0;
        for i in range (0, len(patterns)):
            if patterns[i] in word:
                res=res+1;
        return res
        

File: 1967-number-of-strings-that-appear-as-substrings-in-word/1967-number-of-strings-that-appear-as-substrings-in-word.cpp
class Solution {
public:    
    int numOfStrings(vector<string>& patterns, string word) {
        int res=0;
        
        for(int i=0;i<patterns.size();i++) {
            int pos = word.find(patterns[i]);
            if(pos>=0) res++;
        }
        
        return res;
    }
};